src/OrchardCore.Modules/OrchardCore.ContentLocalization/Liquid/RemoveCultureFilter.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Fluid;
+using Fluid.Values;
+using OrchardCore.ContentLocalization.ViewModels;
+using OrchardCore.Liquid;
+
+namespace OrchardCore.ContentLocalization.Liquid
+{
+    public class RemoveCultureFilter : ILiquidFilter
+    {
+        public ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArguments arguments, TemplateContext ctx)
+        {
+            var cultureName = arguments["cultureName"]?.ToStringValue();
+            if (string.IsNullOrEmpty(cultureName))
+            {
+                throw new ArgumentException("(remove_culture_by_name) You must supply a cultureName argument");
```

Comment:
Do we need **(remove_culture_by_name)** in exception messages?!!

Reply:
I dislike to be at the beginning of the error message, but let us see the folks feedback

Reply:
I see